amgut/db/patches/0032.sql

```diff
@@ -0,0 +1,369 @@
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Holy See (Vatican City State)', 96);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Honduras', 97);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Hong Kong', 98);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Hungary', 99);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Iceland', 100);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'India', 101);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Indonesia', 102);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Iran, Islamic Republic of', 103);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Iraq', 104);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Ireland', 105);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Isle of Man', 106);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Israel', 107);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Italy', 108);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Jamaica', 109);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Japan', 110);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Jersey', 111);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Jordan', 112);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kazakhstan', 113);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kenya', 114);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kiribati', 115);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Korea, Democratic People''s Republic of', 116);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Korea, Republic of', 117);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kuwait', 118);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kyrgyzstan', 119);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Lao People''s Democratic Republic', 120);
```

Comment:
I'm not familiar enough with postgres.  Is this the is `''` the proper way to do escape characters?